client/app/locales/en.coffee

```diff
@@ -137,6 +139,7 @@ module.exports =
     'konnector description podcast': 'Download your favourite audio podcasts from a RSS feed. This import can take a while.'
     'konnector description materiel_net': 'Import your Materiel.net bills. This konnector requires the Files application to store the bill PDF files.'
     'konnector description meetup': 'Synchronize your Meetup calendar with your Cozy. This konnector requires the Calendar application.'
+    'konnector description maif': "Maif invites you to download some data in your Cozy (societary, home, claims, ...). To do that, (1) use the 'connect' button to connect on your Maif account, (2) select 'every day' as update schedule, then (3) click on 'Import and Save'."
```

Comment:
Ok for the ol, it will be cleaner.
We advice daily update because update on the data could happen, and we which that the user could look into it's Cozy to view them (not ve forced to go back to some other Maif interface to view the uptodate informations). But, as the user get more experienced, he will understand that he have the control on this setting.